src/main/java/com/teamscale/upload/CommandLine.java

```diff
@@ -315,9 +331,11 @@ private void validateKeystoreSettings(ArgumentParser parser) throws ArgumentPars
 
 	private void validateAccessKey(ArgumentParser parser) throws ArgumentParserException {
 		if (accessKey == null) {
-			throw new ArgumentParserException("You did not specify a Teamscale access key. You can either specify"
-					+ "it via --accesskey <access key>, via setting the environment variable $TEAMSCALE_ACCESSKEY or via stdin"
-					+ " using '--accesskey -'.", parser);
+			throw new ArgumentParserException(
+					"You did not specify a Teamscale access key. You can either specify"
+							+ "it via --accesskey <access key>, via setting the environment variable $"
+							+ TEAMSCALE_ACCESS_KEY_ENVIRONMENT_VARIABLE + " or via stdin" + " using '--accesskey -'.",
```

Comment:
I think the last string concatenation got mixed up by the change and can be removed. Will do this in my rework